lib/traject/macros/marc21_semantics.rb

```diff
@@ -364,13 +364,18 @@ def self.publication_date(record, estimate_tolerance = 15, min_year = 500, max_y
           end
         end
       end
-      # Okay, nothing from 008, try 260
+      # Okay, nothing from 008, first try 264, then try 260
       if found_date.nil?
+        v264c = MarcExtractor.cached("264|*1|c", :separator => nil).extract(record).first
         v260c = MarcExtractor.cached("260c", :separator => nil).extract(record).first
         # just try to take the first four digits out of there, we're not going to try
         # anything crazy.
-        if m = /(\d{4})/.match(v260c)
+        if m = /(\d{4})/.match(v264c)
           found_date = m[1].to_i
+        else
+          if m = /(\d{4})/.match(v260c)
+            found_date = m[1].to_i
+          end
```

Comment:
Minor stylistic thing, but can you change this to `elsif` without the separate inner block? I think that would do the same thing? This is already convoluted logic, so let's try to keep it as simple as we can!